dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/appmanager/JCloudsAppStorageService.java

```diff
@@ -261,14 +261,12 @@ public App installApp( File file, String filename )
         App app = new App();
         log.info( "Installing new app: " + filename );
         
-        try
+        try( ZipFile zip = new ZipFile( file ) )
```

Comment:
Remember to actually test this part well ;)

Reply:
tested